core/templates.yml

```diff
@@ -67,6 +67,10 @@ keys:
     SEQ:
         type: sequence
         format_spec: "04"
+    timecode:
+        type: sequence
```

Comment:
A timecode can mean so many different things - can you please add some detailed comments of what it means in this context, along with an example of what it looks like - just to avoid any confusion!

I think normally, for our clients a time code is a SMPTE time code, e.g HH:MM:SS:FF - i think, just to avoid confusion, it would be better if we called it `flame.timecode` rather than just `timecode`